### PR TITLE
Can disabled or enable toolbox from grid!

### PR DESCRIPTION
**Pull request by davisvagris**
Can disabled or enable toolbox (three dots) from the grid! Default toolbox is enabled. 

Code
`$grid->toolbox(false/true)`

### CODE REVIEW

File: src/Admin/Grid.php [hunk 1]
```diff
@@ -70,6 +70,11 @@ class Grid
      */
     protected $rowUrlCallback;
 
+    /**
+     * @var bool
+     */
+    protected $toolbox = true;
```
Comment (exabyssus) [inline src/Admin/Grid.php]: It would be more descriptive to call this attribute "hasToolbox" or "showToolbox"

File: src/Admin/Grid.php [hunk 2]
```diff
@@ -147,6 +152,26 @@ public function addTool(RenderableInterface $tool, string $side = null)
         $this->tools[] = [$tool, $side ?: self::FOOTER_SIDE_SECONDARY];
     }
 
+    /**
+     * @param  bool  $show
+     *
+     * @return \Arbory\Base\Admin\Grid
+     */
+    public function toolbox(bool $show): self
```
Comment (exabyssus) [inline src/Admin/Grid.php]: Consider using showToolbox / hideToolbox methods instead.

**Comment by davisvagris** [comment]
Changed!

File: src/Admin/Grid.php [hunk 3]
```diff
@@ -147,6 +152,34 @@ public function addTool(RenderableInterface $tool, string $side = null)
...
+    {
+        $this->hasToolbox = false;
+
+        return $this;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isToolboxEnable(): bool
```
Comment (rerzy) [inline src/Admin/Grid.php]: should be named `isToolboxEnabled()`

**Pull request merged**